src/main/java/frc/robot/subsystems/Wrist.java

```diff
@@ -113,5 +102,6 @@ public boolean isAtTarget() {
   public void periodic() {
     // This method will be called once per scheduler run
     Logger.recordOutput("Wrist Enc Pos", encoder.getPosition());
+    Logger.recordOutput("Wrist Abs Enc Pos", encoder.getAbsPosition());
```

Comment:
Let's also add logging for the motor position. That'll help debug issues where the motor and the encoder are out of sync